File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int i=0,j=1;
        int ans =1;
        while(j<nums.size()){
            if(nums[i]==nums[j]){
                j++;
            }
            else{
                i++;
                ans++;
                nums[i] = nums[j];
            }
        }
        return  ans;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        i=0
        j=0
        ans=1
        for j in range(0,len(nums)):
            if nums[i]!=nums[j]:
                i+=1
                ans+=1
                nums[i],nums[j]=nums[j],nums[i]


        return ans

        